### PR TITLE
I was testing you and you passed.

### PR DESCRIPTION
**Pull request by MCorder00**
My code is ready for prime time. We are ready to rock. Kinda sorta.

### CODE REVIEW

File: Group_Hotel/src/main/java/org/example/Main.java [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("I am a line of code. My hovercraft is full of eels.");
```
Comment (herbcerr) [inline Group_Hotel/src/main/java/org/example/Main.java:5]: I agree.

Comment (MCorder00) [reply Group_Hotel/src/main/java/org/example/Main.java:5]: I am glad that you agree.

Comment (AstorAlina) [inline Group_Hotel/src/main/java/org/example/Main.java:5]: Heya

Comment (sunwarpradeep) [reply Group_Hotel/src/main/java/org/example/Main.java:5]: I agree as well

**Review by AstorAlina — APPROVED** [review]

**Review by herbcerr — APPROVED** [review]

**Pull request merged**